tests/misc_env/install_prereq.py

```diff
@@ -171,12 +171,17 @@ def install_prereq(
         if skip_enabling_rhel_rpms and skip_subscription:
             # Ansible is required for RHCS 4.x
             if distro_ver.startswith("8"):
+                # TODO(vamahaja): Temporary changes. Revert ansible package with latest epel repo.
+                ansible_pkg = (
```

Comment:
Our tests are executed in two environments. This would work for environments within RH network. I believe it would be good if we have a check for it also. What do you think?

Reply:
@psathyan this is temporary fix specific to interop test run.  Adding support for multiple platform will unnecessarily complicates code. We are already working on to get proper repo to install required ansible version, will revert changes once done.